lib/docpub.js

```diff
@@ -49,8 +51,8 @@ module.exports = class DocpubPipeline {
 
         return category.read()
             .then(() => {
-                const categoryUploader = new CategoryUploader(category);
-                const zendeskDeleter = new ZendeskDeleter(category);
+                const categoryUploader = new CategoryUploader(category, this._config);
```

Comment:
I thought about it, but decided to pass config because:

1. I think it would be better to keep information about current API client as private detail of it's implementation. For client it's better to be not aware about network layer.
2. It would be needed to pass config to uploaders anyway. For instance, if we required to implement feature about force upload from backlog, we need to provide config to client.